<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payroll</class>
 <widget class="QMainWindow" name="Payroll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payroll</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pngs.qrc">
    <normaloff>:/downloads/newicons/cash_register.png</normaloff>:/downloads/newicons/cash_register.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Payroll Month:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Recalculate">
        <property name="text">
         <string>Recalculate</string>
        </property>
        <property name="icon">
         <iconset resource="pngs.qrc">
          <normaloff>:/downloads/newicons/cash_register.png</normaloff>:/downloads/newicons/cash_register.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdClear">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset resource="rc.qrc">
          <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Payroll Calculation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CustomQueryView" name="lstPayroll">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Payroll Lists</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Bank Lists</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="CustomQueryView" name="trvBankLists"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>NSSF List</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="cmdExportNSSF">
                <property name="text">
                 <string>Generate Excel File</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextBrowser" name="txtNSSFView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>NHIF List</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="cmdExportNHIF">
                <property name="text">
                 <string>Generate Excel File</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextBrowser" name="txtNHIFView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>P10A</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Select Year:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cboP1AYears"/>
              </item>
              <item>
               <widget class="QPushButton" name="cmdShowP10A">
                <property name="text">
                 <string>Show P10A</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PayrollReportPreview" name="widgetP10A" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_8">
           <attribute name="title">
            <string>P10D</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Select Year:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cboP1DYears"/>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Quarter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cboP10DQuarters">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Q1 (Jan, Feb, Mar)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Q2 (Apr, may, Jun)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Q3 (Jul, Aug, Sep)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Q4 (Oct, Nov, Dec)</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cmdShowP10D">
                <property name="text">
                 <string>Show P10D</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PayrollReportPreview" name="widgetP10D" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_9">
           <attribute name="title">
            <string>P9A</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Select Year:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cboP9AYears"/>
              </item>
              <item>
               <widget class="QPushButton" name="cmdShowP9A">
                <property name="text">
                 <string>Show P9A</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PayrollReportPreview" name="widgetP9A" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpPayrollProgress">
      <property name="title">
       <string>Calculating Payroll...</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblPayrollProgressData">
         <property name="text">
          <string>Calculating...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="prgPayrollProgress">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
  <customwidget>
   <class>PayrollReportPreview</class>
   <extends>QWidget</extends>
   <header>payrollreportpreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
  <include location="pngs.qrc"/>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
